controllers/AcquisitionSessionController.php

```diff
@@ -0,0 +1,291 @@
+<?php
+//******************************************************************************
+//                                       AcquisitionSessionController.php
```

Comment:
This object is just for file could rename your controller to AcquisitionSessionMetadataFileController

Reply:
Done